Assignments/stu1/assignments.py

```diff
@@ -103,4 +102,6 @@ def ex7():
 #
 # Place your functions here...
 #
-
+def sort_people(people_list, x, y):
```

Comment:
`y` is passed into this function, but it's not being used.  I think we need to use it to handle the sort direction.

Reply:
Well, no.  What you had before was more correct.  We need to be able to support sorting the list in asc or desc order.  We need an if statement in this function.